packages/backend/src/tokens/tokenTypes.ts

```diff
@@ -0,0 +1,11 @@
+export const TokenType = {
+  SessionToken: 'session_token',
+  ApiKey: 'api_key',
+  MachineToken: 'machine_token',
+  OAuthToken: 'oauth_token',
+} as const;
```

Comment:
This `TokenType` const will be used across the codebase, instead of manually using string literals in conditions and types. Prevents typos and token type changes maintainable.